src/pages/signin/index.js

```diff
@@ -0,0 +1,170 @@
+import React, { Component } from 'react'
+import GoogleLogin from 'react-google-login'
+import { connect } from 'react-fetches'
+
+import ButtonSignin from '../../components/atoms/button-signin'
+import TextFieldGroup from '../../components/molecules/text-field-group'
+import PasswordField from '../../components/molecules/password-field'
+import styles from '../signin/styles.css'
+import FacebookLogin from '../../components/organisms/facebook-login'
+import Cookie from 'js-cookie'
+import Router from 'next/router'
+import { signOut } from '../../utils/SignOut'
+import Button from '../../components/atoms/button'
+import { protectedRouter } from '../../hoc/with-auth'
+
+const mapDispatchToProps = (http, dispatch) => ({
+  submitHandler: dispatch(http.post('login')),
+  submitFacebookHandler: dispatch(http.post('social/facebook')),
+  submitGoogleHandler: dispatch(http.post('social/google')),
+})
+
+class SignIn extends Component {
+  state = {
+    loggedUser: {},
+    username: '',
+    password: '',
+    token: '',
+    loginError: '',
+    authenticated: Cookie.get('token') ? true : false,
+  }
+
+  storeToken = token => {
+    Cookie.set('token', token)
```

Comment:
Same here, maybe the token could be a env var and this could be added to a util file.